compiler/src/iree/compiler/Codegen/Dialect/GPU/IR/IREEGPUOps.cpp

```diff
@@ -185,4 +189,86 @@ void BufferResourceCastOp::getCanonicalizationPatterns(
   results.add<FoldBufferCastOfTensorCast>(ctx);
 }
 
+//===----------------------------------------------------------------------===//
+// CoalescedGatherDMAOp
+//===----------------------------------------------------------------------===//
+
+// DestinationStyleOpInterface implementation
+MutableOperandRange CoalescedGatherDMAOp::getDpsInitsMutable() {
+  return getInitMutable();
+}
+
+LogicalResult CoalescedGatherDMAOp::verify() {
+  // Verify that this op must be inside a scf.forall op
+  Operation *parent = (*this)->getParentOp();
+  bool insideForall = false;
+  while (parent) {
```

Comment:
... this is just `getParentOfType<scf::ForallOp>`, no?

Reply:
yup.